mwsql-blogpost.ipynb

```diff
@@ -0,0 +1,557 @@
+{
```

Comment:
<p>Instead of "please refer to" just say  "see the mwsql documentation".</p><p> </p><blockquote>Ideally, you would never run into encoding errors, but some dump files contain unusual characters.  If the parser complains [todo: would be good to be more specific here and instead give an example of what the parser will say when it lets you know that there's an encoding issue], you may need to try setting a different encoding, such as <code>latin-1</code>. </blockquote><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/blancadesal/wm-tutorials-and-blogposts/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Done</p>